src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -856,6 +885,42 @@ export const getOrganisations = (operatorData: Operator[]): NetexOrganisationOpe
             $t: getNetexMode(operator.mode),
         },
     }));
+    if (baseOperatorInfo) {
+        organisations.push({
+            version: '1.0',
+            id: `noc:${baseOperatorInfo.schemeOperatorName}-${baseOperatorInfo.schemeOperatorRegionCode}`,
+            PublicCode: {
+                $t: `${baseOperatorInfo.schemeOperatorName}-${baseOperatorInfo.schemeOperatorRegionCode}`,
+            },
+            Name: {
+                $t: baseOperatorInfo.schemeOperatorName,
+            },
+            ShortName: {
+                $t: baseOperatorInfo.schemeOperatorName,
```

Comment:
Are these going to come out as null if they arent there? Unsure if we need to add the || '' stuff on the end - theres a chance the netex generator will do that automatically if its null, im not sure.